datura/tools/twitter/twitter_toolkit.py

```diff
@@ -12,9 +13,10 @@ class TwitterToolkit(BaseToolkit, ABC):
     toolkit_id = "0e0ae6fb-0f1c-4d00-bc84-1feb2a6824c6"
 
     def get_tools(self) -> List[BaseTool]:
-        return [TwitterSearchTool()]
+        return [TwitterSearchTool(), TwitterAdvancedSearchTool()]
```

Comment:
these tools have the same slug and uid

Reply:
Is not finished yet, just run in development mode